src/AuthorSection.js

```diff
@@ -46,31 +46,26 @@ AuthorSection.propTypes = {
   fixed: PropTypes.bool,
   optionalComponent: PropTypes.any,
   reverseColor: PropTypes.bool,
+  neutral: PropTypes.bool,
 };
 
 const styles = StyleSheet.create({
   container: {
-    flexDirection: 'row',
     justifyContent: 'flex-start',
```

Comment:
We shouldn't delete `flexDirection: row` and other flex styles in order to keep the same layout for this component.